### ISSUE TITLE
Maybe move workflow specific job details to a new page in the app?

### ISSUE THREAD

**Issue by sckott** [FredHutch/shiny-cromwell#123]
see discussion starting at https://github.com/FredHutch/shiny-cromwell/pull/120#issuecomment-2292348057

What do folks think about this? @vortexing @seankross @sitapriyamoorthi

**Comment by tefirman** [comment]
I believe this will implemented in the GUI update to bslib, so I'm setting this as v1.2, but @sckott @codebeaker, let me know if that's incorrect.

**Comment by sckott** [comment]
Yes this is in the works in bslib branch

**Comment by seankross** [comment]
Yeah let's just make sure `Closes [this issue]` makes its way into the right PR.

**Comment by sckott** [comment]
I don't follow, sorry 🙏🏽 can you explain further @seankross

**Comment by seankross** [comment]
To clarify:

> I guess leave open until we merge to main?

Yes, I agree. When we open a PR that will merge this change into main, let just make sure we mention that the PR resolves this issue.

**Comment by sckott** [comment]
Got it
